sources/packages/web/src/views/student/appeal/StudentAppealSharedFormSubmission.vue

```diff
@@ -174,6 +182,14 @@ export default defineComponent({
       { immediate: true },
     );
 
+    // Clear selected appeals when selected application changes.
+    watch(
+      () => selectedApplicationId.value,
+      () => {
+        selectedApplicationAppeals.value = undefined;
+      },
+    );
```

Comment:
Is there a reason this is not taken care of inside the computed `applicationAppeals`?